sig-advocacy-and-outreach/summer-ospp/README.md

```diff
@@ -11,11 +11,22 @@ KubeSphere in [Summer 2021 of Open Source Promotion Plan](https://github.com/sum
 
 ## Students
 
+Please check out the following pages if you're a student who is interested in this program.
+
 * [Students application process](students-application-process.md)
 * [Open Source Promotion Plan Portal](https://summer.iscas.ac.cn/)
 
+## Administrators
+
+The following people are the administrators of this program in 2021. Please contact them if you have any questions.
+
+* [shenhonglei](https://github.com/shenhonglei)
```

Comment:
hi @shenhonglei , please let me know what's your preferred name on this page.

Reply:
shenhonglei